src/androidTest/java/com/owncloud/android/GetCapabilitiesTest.java

```diff
@@ -82,4 +81,45 @@ public void testGetRemoteCapabilitiesOperationEtag() {
             assertEquals(capability.getEtag(), sameCapability.getEtag());
         }
     }
+
+    /**
+     * Test get capabilities
+     */
+    @Test
+    public void testGetRemoteCapabilitiesOperationWithNextcloudClient() {
+        // get capabilities
+        RemoteOperationResult result = new GetCapabilitiesRemoteOperation().execute(nextcloudClient);
+        assertTrue(result.isSuccess());
+        assertTrue(result.getData() != null && result.getData().size() == 1);
+
+        OCCapability capability = (OCCapability) result.getData().get(0);
+
+        Assert.assertSame(capability.getRichDocuments(), CapabilityBooleanType.FALSE);
+
+        Assert.assertFalse(capability.getDirectEditingEtag().isEmpty());
+
+        // TODO assert basic capabilities
```

Comment:
Just a reminder for future work to be done or an open issue that you want to implement before merging this PR?

Reply:
Better do it now :)